src/eduid_common/api/testing.py

```diff
@@ -308,16 +308,22 @@ def _check_api_response(
         :param payload: Data expected to be found in the 'payload' of the response
         """
         try:
-            assert status == response.status_code, f'The HTTP response code was not {status}'
-            assert type_ == response.json['type'], 'Response had unexpected type'
+            assert status == response.status_code, f'The HTTP response code was {response.status_code} not {status}'
+            assert (
+                type_ == response.json['type']
+            ), f'Wrong response type. expected: {response.json["type"]}, actual: {type_}'
```

Comment:
expected and actual in the format string looks backwards

Reply:
I can only agree but that is the way `assert expected == actual` and `self.assertEqual(expected, actual)` works.